examples/istioctl-analyze/.expected/config.yaml

```diff
@@ -1 +1,2 @@
 exitCode: 1
+runTimes: 2
```

Comment:
IIRC we are using the test harness from the kpt repo after the migration. We can change the name in the kpt repo.
The default is 1, since kpt doesn't need to run everything twice.

Reply:
made a PR for this: #1471

Reply:
Updated the driver version.